Data/Localizations/ru.lua

```diff
@@ -32,7 +33,7 @@ L.AUTO = "Авто";
 L.AUTO_AWARD = "Авто награда";
 L.AUTO_TRADE_OPTIONS = "Авто обмен";
 L.AVAILABLE_PLACEHOLDER_VALUES = "Доступные значения:";
-L.AWARD = "Награда";
+L.AWARD = "   Отдать   "; -- KartonArmadon
```

Comment:
Same as with Start / Stop: is the padding necessary?

Reply:
Not really... It just make buttons on "alt+left_mouse_button" (single item auction) menu almost same width